overlay/lib.cpp

```diff
@@ -442,6 +442,48 @@ extern "C" __declspec(dllexport) unsigned int __cdecl GetOverlayMagicVersion() {
 	return OVERLAY_MAGIC_NUMBER;
 }
 
+// Via d3d9.cpp
+extern "C" __declspec(dllexport) void __cdecl PrepareD3D9();
```

Comment:
Now that it is called from the helper exe which loads the DLL, we don’t have to export it any more, right?
Is it still exported for backwards compatibilty? (this dll with older Mumble version?)
Should work as apart from this declaration only client code was moved.